src/components/structures/AutoHideScrollbar.js

```diff
@@ -111,6 +123,9 @@ export default class AutoHideScrollbar extends React.Component {
 
     render() {
         installBodyClassesIfNeeded();
+        if (this._needsOverflowListener === null) {
```

Comment:
Maybe these one-time setup tasks would be easier to read and avoid extra checks if they were outside of `render`, such as in `componentDidMount`?

Reply:
That is a very good point.